pages/dkp/kaptain/2.0.0/upgrade/index.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: layout.pug
+navigationTitle: Upgrade
+title: Upgrade Kaptain
+menuWeight: 7
+excerpt: Upgrade Kaptain on your cluster
+beta: false
+enterprise: false
+---
+
+Upgrade a Kaptain 1.x installation to 2.0.0
+
+## Background
+
+Kaptain 2.0.0 (and later) are installed with Helm, rather than Kudo. As a result, a upgrading requires that you fully remove your prior Kaptain installation (including deployed models, profiles, notebooks, pipelines, etc.), and install Kaptain 2.0.0 fresh. As model artifacts are stored in your object store, you will not lose these so long as your model store is not deployed to the Kaptain namespace (I.E. you are not using the Minio server).
+
+## Prerequisites
+
+Before you begin, ensure you have:
+
+- The Helm CLI downloaded and installed.
+- The Kudo CLI downloaded and installed.
+- A Linux terminal with kubectl configured to your cluster.
+- The existing cluster meets the criteria listed in the [installation documentation][install]. 
+
+## Upgrade Kaptain
+
+* Ensure the following base addons that are needed by Kaptain are enabled in your Konvoy cluster:
+    ```yaml
+    - configRepository: https://github.com/mesosphere/kubernetes-base-addons
+      configVersion: stable-1.20-4.3.0
+      addonsList:
+        - name: istio
+          enabled: true
+        - name: dex
+          enabled: true
+        - name: cert-manager
+          enabled: true
+        - name: prometheus
+          enabled: true
+    ```
+
+* Ensure the Kaptain addon repository is present in your Konvoy `cluster.yaml`:
+  ```yaml
+      - configRepository: https://github.com/mesosphere/kubeaddons-kaptain
+        configVersion: stable-1.20-1.4.0
+        addonsList:
+          - name: knative
+            enabled: true
+  ```
+
+* [Download][download] the `kubeflow-2.0.0.tgz` tarball from the support website.
+
+### Fully Uninstall Kaptain
```

Comment:
I'd suggest we move this part to 1.3 docs and make it a dedicated docs "uninstall Kaptain". that should make it easier vor everyone to deal with Konvoy 1.x vs DKP 2.x confusions. (Kaptain 2.0.0 will not be supported on Konvoy 1.x)